schemes/chemistry/prescribed_aerosols_namelist.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl"?>
+
+<entry_id_pg version="2.0">
+  <entry id="prescribed_aero_datapath">
+    <type>char*256</type>
+    <category>chemistry</category>
+    <group>prescribed_aero_nl</group>
+    <standard_name>datapath_for_prescribed_aerosols</standard_name>
+    <units>none</units>
+    <desc>
+      Full pathname of the directory that contains the files specified in prescribed_aero_filelist.
+    </desc>
+    <values>
+      <!-- bam: ${DIN_LOC_ROOT}/atm/cam/chem/trop_mozart_aero/aero -->
+      <!-- mam: ${DIN_LOC_ROOT}/atm/cam/chem/trop_mam/aero -->
+      <value>${DIN_LOC_ROOT}/atm/cam/chem/trop_mozart_aero/aero</value>
+    </values>
+  </entry>
+
+  <!-- TODO this was used in build-namelist for setting aerodep_flx_specifier
+       and prescribed_aero_specifier.
+       But namelist options cannot depend on other namelist options in SIMA, I think?
+       How this can be well used needs to be studied -->
```

Comment:
Namelist variables can be used to control other namelist variables in SIMA, it just takes some modifications on the CAM-SIMA side.  You can see an example of this being done for the `pver` to `nlev` attribute setting in CAM-SIMA here:

https://github.com/ESCOMP/CAM-SIMA/blob/development/cime_config/buildnml#L338